meta-mender-commercial/recipes-mender/mender-client/mender-client_%.bbappend

```diff
@@ -0,0 +1 @@
+# Deliberately empty bbappend file to remove bitbake warnings about no bb files
```

Comment:
In a fresh setup of [Rpi4](https://hub.mender.io/t/raspberry-pi-4-model-b/889) on Zeus, I get the following:
```
$ export BRANCH=zeus
$ repo init -u https://github.com/mendersoftware/meta-mender-community.git \
>            -m meta-mender-raspberrypi/scripts/manifest-raspberrypi.xml \
>            -b ${BRANCH}
$ repo sync
$ source setup-environment raspberrypi
You had no conf/local.conf file. This configuration file has therefore been
created for you with some default values. You may wish to edit it to, for
example, select a different MACHINE (target hardware). See conf/local.conf
for more information as common configuration options are commented.

You had no conf/bblayers.conf file. This configuration file has therefore been
created for you with some default values. To add additional metadata layers
into your configuration please add entries to conf/bblayers.conf.

The Yocto Project has extensive documentation about OE including a reference
manual which can be found at:
    http://yoctoproject.org/documentation

For more information about OpenEmbedded see their website:
    http://www.openembedded.org/


### Shell environment set up for builds. ###

You can now run 'bitbake <target>'

Common targets are:
    core-image-minimal
    core-image-sato
    meta-toolchain
    meta-ide-support

You can also run generated qemu images with a command like 'runqemu qemux86'

Other commonly useful commands are:
 - 'devtool' and 'recipetool' handle common recipe tasks
 - 'bitbake-layers' handles common layer tasks
 - 'oe-pkgdata-util' handles common target package tasks

$ bitbake-layers add-layer /work/dmoseley/scratch/mender-new/sources/meta-mender/meta-mender-commercial/
NOTE: Starting bitbake server...
<dmoseley@aruba> {/work/dmoseley/scratch/mender-new/build}
$ bitbake core-image-full-cmdline
Parsing recipes: 100% |############################################################################################################################################| Time: 0:00:08
Parsing of 2369 .bb files complete (0 cached, 2369 parsed). 3503 targets, 126 skipped, 0 masked, 0 errors.
WARNING: No bb files matched BBFILE_PATTERN_mender-commercial '^/work/dmoseley/scratch/mender-new/sources/meta-mender/meta-mender-commercial/'
NOTE: Resolving any missing task queue dependencies

Build Configuration:
BB_VERSION           = "1.44.0"
BUILD_SYS            = "x86_64-linux"
```

The alternative to this patch for that warning is to modify layer.conf and remove bbappends but I'm worried that we may have cause to add some bbappends in the future and it will be tricky to understand why they are not applied.  Let me know how you think we should solve this @lluiscampos

Reply:
Thank you, I am now finally able to understand the problem (and to reproduce it!).

I think your suggested alternative, to just remove bbappens from `conf/layer.conf`, is a better idea. This is a very small layer that we will always have tight control on. If we ever are adding a `.bbapend` file for other recipes we will for sure notice it not being applied.

Adding this dummy file, to me, seems to indicate that `mender-client` directory is meant to be kept here for the user to add content or something.

Reply:
@lluiscampos I updated all the commits to remove the bbappend from BBFILES.